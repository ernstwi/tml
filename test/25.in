init foo

default edge
    reset { x }

default edge
    reset { x }

default location invar { x < 5 }
default location invar { x < 5 }
