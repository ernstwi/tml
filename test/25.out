Semantic error: 2 location defaults
